core/templates/pages/contributor-dashboard-page/services/contribution-and-review-backend-api.service.ts

```diff
@@ -101,11 +111,15 @@ export class ContributionAndReviewBackendApiService {
     }
     if (fetchType === this.REVIEWABLE_QUESTION_SUGGESTIONS) {
       return this.fetchReviewableSuggestionsAsync(
-        'skill', 'add_question', limit, offset);
+        'skill',
+        'add_question',
+        {limit: limit, offset: offset, sortKey: sortKey});
```

Comment:
Can these be passed as separate args (using null in the case where appropriate)? That is better than passing mutable dictionaries/objects around.